js/webform_copy_value.js

```diff
@@ -17,18 +21,24 @@ jQuery(document).ready(function ($) {
   $("input[type=checkbox]").change(function() {
     copyValueToTargetCheckbox(this, sourceTargetMap);
   });
-  
-  $("input[type=radio]").change(function() {
+
+  $("input[type=radio]").click(function() {
```

Comment:
Just wondering if click() action is enought to catch radio change? Maybe use change() action too?